sdks/python/apache_beam/runners/interactive/interactive_environment.py

```diff
@@ -0,0 +1,106 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Module of the current Interactive Beam environment.
+
+Provides interfaces to interact with existing Interactive Beam environment.
+Internally used by Interactive Beam. External Interactive Beam users please use
```

Comment:
For marking internal use, we generally use "For internal use only; no backwards-compatibility guarantees." on its own line.

See: https://github.com/apache/beam/blob/51a13f0a8e5a1088f8a96942b4787d7da25e54dc/sdks/python/apache_beam/utils/counters.py#L23

Reply:
Applied the convention.